src-plugins/itkFilters/itkFiltersToolBox.cpp

```diff
@@ -106,13 +101,6 @@ itkFiltersToolBox::itkFiltersToolBox ( QWidget *parent ) : medFilteringAbstractT
 
     QObject::connect ( d->filters, SIGNAL ( currentIndexChanged ( int ) ), this, SLOT ( onFiltersActivated ( int ) ) );
 
-    QLabel * dataTypeLabel = new QLabel ( tr ( "Data type :" ) );
-    d->dataTypeValue = new QLabel ( tr ( "Unknown" ) );
-
-    QHBoxLayout * dataTypeLayout = new QHBoxLayout;
-    dataTypeLayout->addWidget ( dataTypeLabel );
-    dataTypeLayout->addWidget ( d->dataTypeValue );
```

Comment:
I actually liked this feature, very helpful for testing, debugging.
Why not removing it from this toolbox, but it would be good no to loose it.
Maybe we can do that in another PR (removing and replacing I mean)